server/.env.example

```diff
@@ -1,4 +1,10 @@
-CORE_DATABASE_URL=postgresql://postgres:postgres@localhost:5432/wise-old-man_LOCAL?schema=public
```

Comment:
https://github.com/wise-old-man/wise-old-man/pull/1471

fixed this separately, but thanks for the heads up

Reply:
reverted